cms/src/components/content/EditContent.vue

```diff
@@ -230,10 +244,20 @@ watch(selectedLanguage, () => {
     >
         <template #actions>
             <LBadge v-if="isLocalChange" variant="warning" class="mr-4">Offline changes</LBadge>
-
-            <LButton type="button" @click="save" data-test="save-button" variant="primary">
-                Save
-            </LButton>
+            <div class="flex gap-1">
+                <LButton
+                    type="button"
+                    @click="revertChanges"
+                    data-test="revert-changes-button"
+                    variant="tertiary"
```

Comment:
How does the `secondary` attribute looks like ? 
I propose you to make the button title something like: `Revert Changes`

Reply:
The ```secondary``` attribute looks better.
In my opinion "Revert Changes" as to just "Revert" takes the attention away from "save". If you hover over the button it does say "Revert Changes" to hint to the user what the button will do. I think this will be enough to indicate to the user and keep it clean.